src/services/suggestion.service.ts

```diff
@@ -11,6 +11,7 @@ export const getSuggestions = async (supabase: SupabaseClient<Database>) => {
       *,
       suggestion_instruments:suggestion_instrument (
         id,
+        description,
```

Comment:
They don't need to be retrieved when getting the whole list of suggestions right?

Reply:
Correct, removed it